dev-client/src/components/SiteCard.tsx

```diff
@@ -78,10 +78,10 @@ export const SiteCard = ({
           name: TEMP_MODIFIED_NAME,
         })}
       </Text>
-      <Box h="16px" />
+      <Box height={16} />
       <Row alignItems="center">
         <StaticMapView coords={site} style={styles.mapView} />
-        <Box w="4" />
+        <Box width="4" />
```

Comment:
good catch, i need to fix this one.

this is one of the weirdest decisions in NativeBase. they are equivalent, _unless_ someone has defined a theme space value with a number for a name, which NativeBase's default theme does do. post migration, the only way to specify a numeric width will be `{4}` and it will always mean 4 pixels. `"4"` and `"4px"` will be disallowed